composer.json

```diff
@@ -58,11 +58,11 @@
     "slevomat/coding-standard": "^8.13",
     "soyuka/contexts": "^3.3",
     "squizlabs/php_codesniffer": "^3.7",
-    "symfony/browser-kit": "6.4.*",
+    "symfony/browser-kit": "^6.4",
```

Comment:
It is actually slightly better to pin to a `6.4.*` version instead of allowing any version of 6.4 and higher but lower than 5 (`^6.4`). That way we can never install a 6.5 version which would probably need maintenance to the app to work right.

As we know there will never be a 6.5 version we can leave it like this. But normally I would prefer to revert this change.

Reply:
Thx, fixed.